internal/pkg/template/workload.go

```diff
@@ -302,54 +302,119 @@ func (v importedEnvVar) Value() string {
 // A Secret represents an SSM or SecretsManager secret that can be rendered in CloudFormation.
 type Secret interface {
 	RequiresSub() bool
+	RequiresImport() bool
```

Comment:
nit: maybe we can just embed the `importable` interface here
```suggestion
	importable
```